config.default.yaml

```diff
@@ -526,6 +526,8 @@ sector:
     set_color_shares: false
     blue_share: 0.40
     pink_share: 0.05
+    production: ["H2 Electrolysis", "SMR", "SMR CC"] # ["Alkaline electrolyzer large", "Alkaline electrolyzer medium", "Alkaline electrolyzer small", "PEM electrolyzer", "SOEC", "Solid biomass steam reforming", "Biomass gasification", "Biomass gasification CC", "Natural gas steam reforming", "Natural gas steam reforming CC", "Coal gasification", "Coal gasification CC", "Heavy oil partial oxidation"] a list of H2 production technologies that can be added
```

Comment:
rather than production can we find another name, e.g. hydrogen_technologies? technologies or another one?